packages/framework/src/Migrations/Version20240725200526.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
this comment is not related to this migration, just that I do not forget about it - I noticed there is an unused method `UploadedFileRelationRepository::getUploadedFileIdsByEntityNameIdAndNameAndUploadedFiles()`

Reply:
fixed